src/content/dataroom.json

```
@@ -0,0 +1,134 @@
+[
+  {
+    "pageTitle": "Safe Data Room - Key Insights into the Safe Ecosystem",
+    "description": "Uncover the impact of the Safe ecosystem in crypto. Explore detailed metrics on USDC storage, CryptoPunks holdings, and transaction volumes. Access comprehensive reports from Dune Analytics, Messari, Nansen, and more.",
+    "image": "/images/Home/meta-image-homepage.jpg",
+    "component": "common/MetaTags"
+  },
+  {
+    "component": "DataRoom/Hero",
+    "title": "Welcome to Safe Data Room",
+    "text": "Explore Mind Bending Sats from Safe Universe"
+  },
+  {
+    "component": "DataRoom/USDCStorage",
+    "title": "We store... <br/>",
+    "text": "<b>of all USDC</b>",
+    "link": {
+      "title": "Dune dashboard",
+      "href": "https://dune.com/queries/3710102/6242478"
+    },
+    "image": {
+      "src": "/images/Dataroom/USDC.png",
+      "alt": "USDC Coin"
+    }
+  },
+  {
+    "component": "DataRoom/CryptoPunks",
+    "title": "CryptoPunks",
+    "text": "Some text thats changing<br/>Confirming the sources of the data<br/>Any other info regarding the stats",
+    "link": {
+      "title": "Dune dashboard",
+      "href": "https://dune.com/queries/3710102/6242478"
+    }
+  },
+  {
+    "component": "DataRoom/VolumeTransferred",
+    "title": "with all time volume <br/> transferred",
+    "text": "and counting...",
+    "link": {
+      "title": "Dune dashboard",
+      "href": "https://dune.com/queries/3710102/6242478"
+    }
+  },
+  {
+    "component": "DataRoom/TransactionsOnChain",
+    "text": "<b>transactions on chain</b> originate from a <b>Safe</b>",
+    "link": {
+      "title": "Dune dashboard",
+      "href": "https://dune.com/queries/3710102/6242478"
+    }
+  },
+  {
+    "component": "DataRoom/ExternalLinksGrid",
+    "title": "Dune Boards",
+    "items": [
+      {
+        "title": "Safe on All chains",
```

If this is the project, "Safe" should be italicised afaiu.

```suggestion
        "title": "Safe on all chains",
```